web/services/lib/propertiesV2.ts

```diff
@@ -0,0 +1,7 @@
+const getPropertiesV2 = async () => {
+  const resp = await fetch("/api/propertiesV2");
+  const data = await resp.json();
+  return data;
+};
+
+export { getPropertiesV2 };
```

Comment:
is this dead code? I dont see it referenced anywhere?

Reply:
this referenced in useGetPropertiesV2 hook